client/src/views/Login.tsx

```diff
@@ -0,0 +1,47 @@
+import { useNavigate } from 'react-router-dom'
+
+export const Login = () => {
+    const navigate = useNavigate()
+    const isSignedIn = () => (localStorage.getItem('email') ? true : false)
+
+    if (isSignedIn()) {
+        localStorage.clear()
+    }
+    const handleLogin = (event: any) => {
+        const email = event?.target.elements.email.value
+        const password = event?.target.elements.password.value
+        localStorage.setItem('email', email)
+        localStorage.setItem('password', password)
+        navigate('/')
+    }
+
+    return (
+        <>
+            <section
```

Comment:
section although nice html semantics, it rarely used nowadays 
You can use div, or Box (from MUI)